reVX/least_cost_xmission/cost_creator.py

```diff
@@ -0,0 +1,304 @@
+            xcc._write_geotiff('out/multipliers.tif', xcc._profile, mults_arr)
+
+        for power_class, capacity in xc.power_classes.items():
+            logger.info(f'Calculating costs for class {power_class} using a '
+                        f'{capacity}MW line')
+            blc_arr = xcc.create_base_line_costs(capacity)
+
+            if not save_to_h5:
+                xcc._write_geotiff(f'out/blc_{capacity}MW.tif',
+                                   xcc._profile, blc_arr)
+
+            costs_arr = blc_arr * mults_arr
+            if save_to_h5:
+                xcc._write_layer(h5_f, f'tie_line_costs_{capacity}MW',
+                                 xcc._profile, costs_arr)
+            else:
+                xcc._write_geotiff(f'out/tif_line_costs{capacity}MW.tif',
+                                   xcc._profile, costs_arr)
+
+    def create_multipliers(self):
+        """
+        Create costs multiplier raster
+
+        Returns
+        -------
+        numpy.ndarray
+            Costs multiplier raster, including slope and land use
+        """
+        mults_arr = np.ones(self._iso_regions.shape, dtype=np.float32)
+        regions_mask = np.full(mults_arr.shape, False, dtype=bool)
+
+        for r_conf in self._iso_mults:
+            iso = r_conf['iso']
+            logger.info(f'Processing multipliers for region {iso}')
+
+            if self._iso_lookup is not None:
+                iso_name = iso
+                iso = self._iso_lookup[iso]
+                logger.debug(f'{iso_name} has id {iso}')
+
+            mask = self._iso_regions == iso
+            regions_mask = regions_mask | mask
+
+            if 'land_use' in r_conf:
+                r_lu = self._land_use[mask]
+                lu_mult = self._create_land_use_mult(r_lu, r_conf['land_use'])
+                mults_arr[mask] = lu_mult
+
+            if 'slope' in r_conf:
+                r_slope = self._slope[mask]
+                slope_mult = self._create_slope_mult(r_slope, r_conf['slope'])
+                mults_arr[mask] = mults_arr[mask] * slope_mult
+
+        # Calculate multipliers for regions not defined in `config`
+        logger.debug('Processing default region')
+        default_mask = ~regions_mask
+        d_mults = self._default_mults
+
+        if 'land_use' in d_mults:
+            rlu = self._land_use[default_mask]
+            lu_mult = self._create_land_use_mult(rlu, d_mults['land_use'])
+            mults_arr[default_mask] = lu_mult
+
+        if 'slope' in d_mults:
+            r_slope = self._slope[default_mask]
+            slope_mult = self._create_slope_mult(r_slope, d_mults['slope'])
+            mults_arr[default_mask] = mults_arr[default_mask] * slope_mult
+
+        # Set water multiplier last so we don't get super high multipliers at
+        # water body boundaries next to steep slopes
+        mults_arr[self._land_use == WATER_NLCD_CODE] = WATER_MULT
+
+        return mults_arr
+
+    def create_base_line_costs(self, capacity):
```

Comment:
Same as above..
silly, but how about `compute_*` instead of create?
Also I think we should add two kwargs here:
`save_layer`
`save_geotiff`
and add the functionality to do the above into this method, then in the class method we would turn save_layer on and pass through the save_geotiff kwarg

Reply:
done